02_configure_host.sh

```diff
@@ -148,8 +150,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
   if [[ "$IMAGE" =~ "://" ]] ; then
     REPOPATH=~/${IMAGE##*/}
     # Clone to ~ if not there already
-    [ -e "$REPOPATH" ] || git clone "$IMAGE" "$REPOPATH"
-    cd "$REPOPATH" || exit
+    if [ -e "$REPOPATH" ]; then
```

Comment:
it can be like this

if dir not exist
  then clone
end if

cd dir
switch branch